semgrep/semgrep/commands/scan.py

```diff
@@ -750,7 +750,10 @@ def scan(
                 )
             else:
                 resolved_configs, config_errors = semgrep.config_resolver.get_config(
-                    pattern, lang, config or [], project_url=get_project_url()
+                    pattern,
+                    lang,
+                    config or [],
+                    project_url=get_project_url(target_sequence),
```

Comment:
@brendongo this is the biggest thing I'm not sure about. What should the behavior be here for the scan subcommand?